ui/src/components/common/utils.ts

```diff
@@ -45,3 +45,9 @@ export const depositSummary = (deposits: DepositInfo[]): string => {
         .map(([label, total]) => `${label}: ${total}`)
         .join(", ");
 }
+
+export function countDecimals(value: number) {
+    if (Math.floor(value) !== value)
+        return value.toString().split(".")[1].length || 0;
```

Comment:
if my split quantity is `1.2000000000000000000000000000000000` will this count for 1 decimal or 34? Not really a big deal as I can always remove the trailing zeros but just wondering

Reply:
the `toString()` method strips trailing zeros, so the count would be 1